BGAnimations/ScreenSelectMusic overlay/profilecard.lua

```diff
@@ -282,11 +283,7 @@ local function generalFrame(pn)
 
 				local diff = steps[pn]:GetDifficulty()
 				local stype = ToEnumShortString(steps[pn]:GetStepsType()):gsub("%_"," ")
-				local meter = math.floor(steps[pn]:GetMSD(getCurRateValue(),1))
-				if meter == 0 then
-					meter = steps[pn]:GetMeter()
-				end
-				meter = math.max(1,meter)
```

Comment:
I want the theme to fallback to the non-MSD values if the MSD happens to be 0 (unranked). so those lines shouldn't have been removed. Dance Single isn't the only mode even though it's the most played for etterna.